docs/serverless/08-04-trigger-function-with-event.md

```diff
@@ -105,17 +105,44 @@ To test if the Trigger CR is properly connected to the Function:
     }
     ```
 
-2. Send an event manually to trigger the Function:
+2.  Send an event manually to trigger the function. In the first example, a [CloudEvent](https://github.com/cloudevents/spec/blob/v1.0/spec.md) is sent directly to the Event Mesh. The second example shows an event which also reaches the Event Mesh, but before that it is translated byt the compatibility layer to the format compliant with the CloudEvents specification. For details on CloudEvents, exposed endpoints, and compatibility layer read about [event processing and delivery](/components/event-mesh/#details-event-processing-and-delivery).
```

Comment:
```suggestion
2.  Send an event manually to trigger the function. In the first example, a [CloudEvent](https://github.com/cloudevents/spec/blob/v1.0/spec.md) is sent directly to the Event Mesh. The second example shows an event that also reaches the Event Mesh, but it is first modified by the compatibility layer to the format compliant with the CloudEvents specification. For details on CloudEvents, exposed endpoints, and compatibility layer read the [event processing and delivery description](/components/event-mesh/#details-event-processing-and-delivery).
```

Reply:
Maybe we could make a tip out of it? `For details on CloudEvents, exposed endpoints, and compatibility layer read the [event processing and delivery description](/components/event-mesh/#details-event-processing-and-delivery).`

Reply:
Maybe we could mention in parenthesis which Kyma versions these cases refer to?